UI/src/view/dynamicAssetForm.js

```diff
@@ -601,6 +605,187 @@
                 '</div>' );
         };
     };
+
+    var DatePeriodField = function(assetTypeConfiguration, field, isDisabled) {
```

Comment:
This new type it's ok on right side form, but when we use a tool to create in more than one link the popup with the form are with the wrong format.